app/javascript/packages/verify-flow/verify-flow-step-indicator.tsx

```diff
@@ -4,35 +4,68 @@ import { t } from '@18f/identity-i18n';
 import AddressVerificationMethodContext from './context/address-verification-method-context';
 import type { AddressVerificationMethod } from './context/address-verification-method-context';
 
+export enum VerifyFlowPath {
+  DEFAULT = 'default',
+  IN_PERSON = 'in_person',
+}
+
 type VerifyFlowStepIndicatorStep =
   | 'getting_started'
   | 'verify_id'
   | 'verify_info'
   | 'verify_phone_or_address'
-  | 'secure_account';
+  | 'secure_account'
+  | 'find_a_post_office'
+  | 'go_to_the_post_office';
 
-/**
- * Mapping of flow form steps to corresponding step indicator step.
- */
-const FLOW_STEP_STEP_MAPPING: Record<string, VerifyFlowStepIndicatorStep> = {
-  document_capture: 'verify_id',
-  password_confirm: 'secure_account',
-  personal_key: 'secure_account',
-  personal_key_confirm: 'secure_account',
-};
+interface VerifyFlowConfig {
+  /**
+   * Sequence of step indicator steps.
+   */
+  steps: VerifyFlowStepIndicatorStep[];
 
-/**
- * Sequence of step indicator steps.
- */
-const STEP_INDICATOR_STEPS: VerifyFlowStepIndicatorStep[] = [
-  'getting_started',
-  'verify_id',
-  'verify_info',
-  'verify_phone_or_address',
-  'secure_account',
-];
+  /**
+   * Mapping of flow form steps to corresponding step indicator step.
+   */
+  mapping: Record<string, VerifyFlowStepIndicatorStep>;
+}
+
+const FLOW_STEP_PATHS: Record<VerifyFlowPath, VerifyFlowConfig> = {
+  [VerifyFlowPath.DEFAULT]: {
+    steps: [
+      'getting_started',
+      'verify_id',
+      'verify_info',
+      'verify_phone_or_address',
+      'secure_account',
+    ],
+    mapping: {
+      document_capture: 'verify_id',
+      password_confirm: 'secure_account',
+      personal_key: 'secure_account',
+      personal_key_confirm: 'secure_account',
```

Comment:
That's correct for now, yes, and would be something we want to change. I'll keep that in mind as I implement the remainder of the work for this ticket, since this pull request was primarily concerned with the first couple steps.

Reply:
We may be able to do something similar to what we did to implement the "pending" status for GPO in the FSMv2 flow, where we use the address proofing component. For example, we could consider the document proofing component to determine whether the default or in-person flow should be shown.

Reply:
Circling back to this: The revisions needed here are included in #6846.